### ISSUE TITLE
Quesite problem with EQL3

### ISSUE THREAD

**Issue by Lainiel** [AeroScripts/QuestieDev#257]
When handing in the quest, it automaticaly skips the continue button right to the complete button. You can't read what the NPC says before you click continue like this.

If I disable either EQL3 or Questie it starts working again and doesn't skip the continue button.

EQL version 3.6.1
Questie version 3.1

**Comment by Muehe** [comment]
Confirmed. Using ShaguQuest. This didn't happen with 3.0, so it should happen somewhere in Questies EQL3 specific code.

**Comment by slicerlucas** [comment]
I'm have to downgrade to v2.67 to fix this problem :(

**Comment by ShaunDreclin** [comment]
Having the same issue using questie and eql3 together

**Comment by jimbishop** [comment]
I have confirmed this is also happening to myself with EQL3 + Questie 3.1

**Comment by Muehe** [comment]
Fix:

```diff
diff --git a/!Questie/Questie.lua b/!Questie/Questie.lua
index e10288c..30945fb 100644
--- a/!Questie/Questie.lua
+++ b/!Questie/Questie.lua
@@ -418,7 +418,6 @@ function Questie:OnEvent(this, event, arg1, arg2, arg3, arg4, arg5, arg6, arg7,
                                        end
                                        QuestieTrackedQuests[hash] = nil;
                                end
-                               CompleteQuest();
                                Questie:AddEvent("CHECKLOG", 0.135);
                        end
                end

```

@Schaka Do you want me to PR this or can you upload yourself? P.S. Did no regression testing at all. Think none is needed due to the similar code fragment in Modules/QuestieQuest.lua (which is not only redundant but also unused if I am not mistaken).

**Comment by jimbishop** [comment]
Thanks @Muehe I'll give this a shot.

**Comment by Schaka** [comment]
@jimbishop if this does the trick for you, I'll mege Muehe's code.
@Muehe I'd like a PR actually. One, because I want to give credit to you, two because I don't actually have any local code and would prefer not breaking any formatting with the web interface.

**Comment by slicerlucas** [comment]
@jimbishop @Muehe The patch works to me! Thank very much \o/

**Comment by Muehe** [comment]
Closed by 71e871a04705870d9a4942565803139fc940e1ed
